addons/mail/static/src/discuss/core/public_web/discuss_sidebar_categories.xml

```diff
@@ -114,22 +114,23 @@
                     <ThreadIcon t-if="thread.channel_type?.includes('chat') or (thread.channel_type === 'channel' and !thread.authorizedGroupFullName)" thread="thread" size="'small'" className="'o-mail-DiscussSidebarChannel-threadIcon position-absolute bottom-0 end-0 p-1 me-n1 mb-n1 d-flex align-items-center justify-content-center rounded-circle bg-inherit'"/>
                     <CountryFlag t-if="thread.anonymous_country" country="thread.anonymous_country" class="'position-absolute o-mail-DiscussSidebarChannel-country border'"/>
                 </div>
-                <span t-if="!store.discuss.isSidebarCompact" class="mx-2 text-truncate" t-att-class="thread.selfMember?.message_unread_counter > 0 and !thread.mute_until_dt ? 'o-item-unread fw-bolder' : 'text-muted'">
+                <span t-if="!store.discuss.isSidebarCompact" class="mx-2 text-truncate" t-att-class="thread.selfMember?.message_unread_counter > 0 and !thread.isMuted ? 'o-item-unread fw-bolder' : 'text-muted'">
                     <t t-esc="thread.displayName"/>
                 </span>
             </button>
-            <div t-if="!store.discuss.isSidebarCompact" class="flex-grow-1"/>
-            <t t-if="!store.discuss.isSidebarCompact" t-call="mail.DiscussSidebarChannel.commands"/>
             <t t-if="!store.discuss.isSidebarCompact">
-                <t t-foreach="indicators" t-as="indicator" t-key="indicator_index" t-component="indicator" t-props="{ thread }"/>
+              <div class="flex-grow-1"/>
+              <t t-call="mail.DiscussSidebarChannel.commands"/>
+              <t t-foreach="indicators" t-as="indicator" t-key="indicator_index" t-component="indicator" t-props="{ thread }"/>
+              <i t-if="thread.isMuted" class="fa fa-fw text-danger fa-bell-slash me-3"/>
```

Comment:
I would remove `text-danger`, this color asks for action to user, which goes against the intent to mute the channel.